### PR TITLE
[PORT] website_sale_add_to_cart: 11.0

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

**Comment by ilmir-k** [comment]
@KolushovAlexandr fix lints

File: website_sale_add_to_cart/doc/index.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+==================================
+ Quick add items to shopping cart
+==================================
+
+Installation
+============
+{Instruction about things to do before actual installation}
```
Comment (yelizariev) [inline website_sale_add_to_cart/doc/index.rst]: default

**Comment by yelizariev** [comment]
update ``.travis.yml``

**Comment by KolushovAlexandr** [comment]
@ilmir-k Only `# -*- coding: utf-8 -*-` errores are remained in LINT.
@yelizariev  In tests i've got a strange error arising after typing a text in an 'input' and click anything else but the 'input'. I need consultation about that because in checkout_store situation is the same.

**Comment by ilmir-k** [comment]
@KolushovAlexandr it is because you removed # -*- coding: utf-8 -*- from the files. Return it back

**Comment by KolushovAlexandr** [comment]
@ilmir-k How i've understood Ivan said he is going to remove this lint error for version 11

**Comment by ilmir-k** [comment]
@KolushovAlexandr ok, you can skip it

**Comment by yelizariev** [comment]
merge upstream to resolve conflicts

**Comment by yelizariev** [comment]
Fix conflicts

**Pull request merged**